source/mainnet/smart-contracts/tutorials/wCCD/wCCD-front-end-set-up.rst

```diff
@@ -4,6 +4,354 @@
+
+    Enter the below ``JSON-RPC`` endpoint into the browser wallet to connect to
+    your local ``JSON-RPC server`` on port 9095.
+
+    .. code-block:: console
+
+        http://127.0.0.1:9095
+
+    In case you run an older pre-MVP wallet, you have to enter the private key and
+    the associated account into the browser wallet similar to the below string. In case you run a newer pre-MVP wallet,
+    you can create a new account with the associated private key in the browser wallet.
+
+    .. code-block:: toml
+
+        74ff83a13ca066298583dcb9151822359fd2e4c9b69c9ca427455da565f6129b,3oLNhuxM7yrf3LrJa3hH5NfocTViGS8Aj2t6YScWNvUq4o2nC
+
+    You completed the browser wallet setup. Check that your account balance is displayed and you have enough
+    CCD to be able to execute transactions.
+
+    .. note::
+        You are connected to a website with your browser wallet when you see the green ``Connected`` button.
+        You can toggle on/off the connection by clicking on the button.
+
+    .. image:: ./images/wCCD_tutorial_14.png
+        :width: 40 %
+
+.. dropdown:: Workflow 2 - Setup browser wallet connected to hosted node (click here)
+
+    You are ready now to build the pre-MVP browser wallet from the source code.
+
+    Clone the repository with the command:
+
+    .. code-block:: console
+
+        $git clone https://github.com/Concordium/concordium-browser-wallet.git
+
+    .. note::
+
+        Depending on the exact commit hash that you used to build your pre-MVP browser wallet, the
+        screenshots might differ. The browser wallet hosts the private keys corresponding
+        to the accounts of the user and a link that points to a ``JSON-RPC server``.
+
+    The next steps are based on the pre-MVP browser wallet from a git commit after 22.8.2022.
+    These newer versions of the pre-MVP browser wallet connect to the ``JSON-RPC wallet proxy``
+    hosted by Concordium which will take care of the Concordium node on behalf of you.
+
+    You can build the browser wallet extension and load it
+    (``dist`` folder from the path `root/packages/browser-wallet`) into the
+    `Chrome browser <https://developer.chrome.com/docs/extensions/mv3/getstarted/#unpacked>`_ as
+    described in the README file of the
+    `browser wallet repo <https://github.com/Concordium/concordium-browser-wallet/tree/main/packages/browser-wallet>`_.
+
+    You are ready now to start the browser wallet by clicking on the Concordium icon at the top right of the
+    ``Chrome`` browser.
+
+    .. image:: ./images/wCCD_tutorial_18.png
+        :width: 100 %
+
+    .. note::
+
+        The puzzle icon at the top right of the ``Chrome`` browser allows you to manage your browser extensions.
+        You can enable pinning of the Concordium browser wallet.
+
+        .. image:: ./images/wCCD_tutorial_13.png
+            :width: 30 %
+
+    Create a new account on testnet by going through the setup steps of the browser wallet.
```

Comment:
How come we only show the create account part in the easy workflow? Isn't that just as relevant in the other section?

Reply:
The create account flow is only available in the newer pre-MVP wallets (with seed phrase generation). The workflow 2 (more advanced) has to be done by important the private keys instead.